### PR TITLE
[Improve][Connector-V2-File] Add the skip header lines parameter such as csv、txt

### LINKED ISSUE #3192: [improve][connector-v2-file]   Add the skip header lines parameter such as csv、txt
### Search before asking

- [X] I had searched in the [feature](https://github.com/apache/incubator-seatunnel/issues?q=is%3Aissue+label%3A%22Feature%22) and found no similar feature requirement.

### Description

[improve][connector-v2-file]   Add the skip header lines parameter such as csv、txt

### Usage Scenario

_No response_

### Related issues

_No response_

### Are you willing to submit a PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by CallMeKingsley97**
[Improve][connector-v2-file] Add the skip header lines parameter such as csv、txt
#3192

## Check list

* [ ] Code changed are covered with tests, or it does not need tests for reason:
* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/incubator-seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/incubator-seatunnel/tree/dev/docs

### CODE REVIEW

File: seatunnel-connectors-v2/connector-file/connector-file-base/src/main/java/org/apache/seatunnel/connectors/seatunnel/file/source/reader/TextReadStrategy.java [hunk 1]
```diff
@@ -47,15 +47,19 @@ public class TextReadStrategy extends AbstractReadStrategy {
     private DateUtils.Formatter dateFormat = DateUtils.Formatter.YYYY_MM_DD;
     private DateTimeUtils.Formatter datetimeFormat = DateTimeUtils.Formatter.YYYY_MM_DD_HH_MM_SS;
     private TimeUtils.Formatter timeFormat = TimeUtils.Formatter.HH_MM_SS;
+    private Long skipHeaderNumber = 0L;
```
Comment (TyrantLucifer) [inline seatunnel-connectors-v2/connector-file/connector-file-base/src/main/java/org/apache/seatunnel/connectors/seatunnel/file/source/reader/TextReadStrategy.java]:
```suggestion
    private long skipHeaderNumber = 0L;
```

File: seatunnel-connectors-v2/connector-file/connector-file-base/src/main/java/org/apache/seatunnel/connectors/seatunnel/file/source/reader/TextReadStrategy.java [hunk 2]
```diff
@@ -47,15 +47,19 @@ public class TextReadStrategy extends AbstractReadStrategy {
...
 
     @Override
     public void read(String path, Collector<SeaTunnelRow> output) throws IOException, FilePluginException {
+        if (pluginConfig.hasPath(BaseSourceConfig.SKIP_HEADER_ROW_NUMBER)) {
```
Comment (TyrantLucifer) [inline seatunnel-connectors-v2/connector-file/connector-file-base/src/main/java/org/apache/seatunnel/connectors/seatunnel/file/source/reader/TextReadStrategy.java]: Treat `skipHeaderNumber` as a class attribute and init it in `setPluginConfig` is better.

Comment (CallMeKingsley97) [reply seatunnel-connectors-v2/connector-file/connector-file-base/src/main/java/org/apache/seatunnel/connectors/seatunnel/file/source/reader/TextReadStrategy.java]: I'll find a time to optimize

**Review by CalvinKirs — COMMENTED** [review]
Thanks, and don't forget to update the doc https://github.com/apache/incubator-seatunnel/tree/dev/docs/en/connector-v2

File: seatunnel-connectors-v2/connector-file/connector-file-base/src/main/java/org/apache/seatunnel/connectors/seatunnel/file/source/reader/TextReadStrategy.java (hunk 2 above)
Comment (CallMeKingsley97) [reply seatunnel-connectors-v2/connector-file/connector-file-base/src/main/java/org/apache/seatunnel/connectors/seatunnel/file/source/reader/TextReadStrategy.java]: OK, I resolved.

**Comment by CallMeKingsley97** [comment]
> Thanks, and don't forget to update the doc https://github.com/apache/incubator-seatunnel/tree/dev/docs/en/connector-v2
Done!😊

**Pull request closed without merging**